aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSampler.java

```diff
@@ -148,10 +149,23 @@ private void getAndUpdateSampler() {
   }
 
   private void scheduleSamplerUpdate() {
-    long delay = pollingIntervalNanos + RANDOM.nextInt(jitterNanos);
+    @SuppressWarnings("OptionalGetWithoutIsPresent") // getAsLong: RANDOM.longs should always provide a value, so we can do an unchecked unwrap here
+    long delay = pollingIntervalNanos + RANDOM.longs(0, jitterNanos).findFirst().getAsLong();
     pollFuture = executor.schedule(this::getAndUpdateSampler, delay, TimeUnit.NANOSECONDS);
   }
 
+  /**
+   * returns the duration until the next scheduled sampler update or null if no next update is scheduled yet
+   */
+  @Nullable
+  public Duration getNextSamplerUpdateScheduledDuration() {
```

Comment:
This is currently only required for the test. Would that be okay?

I don't have an idea how to test this properly because we don't want to wait for 5 minutes in the test, right? As only long polling intervals trigger this bug so we should use a long polling interval.

Reply:
Hmmm I think the maintainers would have final say but I don't think we want to expose additional public methods in the sampler like this. Is it possible to mock the system clock and trick the test into thinking 5 minutes have passed? Or just otherwise verifying the interval isn't negative when set to 5 minutes?

Reply:
can it be package-private and still accessed from test?

Reply:
Good point, I made the method package-private.